test/append.js

```diff
@@ -6,16 +6,23 @@ var Stream = require( 'stream' )
 
 describe( 'Pipeline.append()', function() {
 
+  it( 'returns pipeline length', function() {
+    var pipeline = new Pipeline()
+    var stream = new Stream.PassThrough()
+    assert.strictEqual( pipeline.append( stream ), pipeline.length )
+    assert.strictEqual( pipeline.length, 1 )
+  })
```

Comment:
Is it worth doing _another_ `append`, and checking that it now returns 2? Otherwise the test would be passed by a method hard-coded to `return 1` ;-)
(and obviously the same for other tested methods too)